test/unit/helpers/dom/EnrollSmsForm.js

```diff
@@ -64,4 +64,4 @@ define(['./Form'], function (Form) {
 
   });
 
-});
```

Comment:
let's revert the change to this file, it is not needed

Reply:
fixed